backend/reviews/handler.js

```diff
@@ -340,12 +346,17 @@ async function writeReviewsToDB(reviews){
  */
 function createDynamoBatchRequest(items) {
   let putItems = [];
+  let putHashes = [];
   for (let i = 0; i < items.length; i++) {
-    putItems.push({
-      PutRequest: {
-        Item: items[i]
-      }
-    });
+    //due to the way empty pages are handled, have to check for duplicate keys created for empty reviews
```

Comment:
Can you explain the thinking behind this code? What is reviewHash set to for empty reviews? I'm not really sure what you mean by "the way empty pages are handled"...